src/tensors/cpu/device.cpp

```diff
@@ -15,18 +15,26 @@ Device::~Device() {
   size_ = 0;
 }
 
+#ifdef _WIN32
```

Comment:
Please add a comment what this is used for, e.g. // allocate function for tensor reserve() below. Needed for AVX2, while not available on all compilers.